doc/languages-frameworks/dotnet.section.md

```diff
@@ -85,20 +87,25 @@ To package Dotnet applications, you can use `buildDotnetModule`. This has simila
 * `dotnetBuildFlags` can be used to pass flags to `dotnet build`.
 * `dotnetTestFlags` can be used to pass flags to `dotnet test`.
```

Comment:
Would you mind adding this comment? Just for consistency, seeing how it's also ran conditionally.
```suggestion
* `dotnetTestFlags` can be used to pass flags to `dotnet test`. Used only if `doCheck` is set to `true`.
```

Reply:
fixed in the updated commit